installer.sh.in

```diff
@@ -493,10 +494,14 @@ at the first 2GB of the disk with the TOGGLE \`bios_grub' enabled.\n
 ${BOLD}NOTE: you don't need this on EFI systems.${RESET}\n\n
 For EFI systems GPT is mandatory and a FAT32 partition with at least\n
 100MB must be created with the TOGGLE \`boot', this will be used as\n
-EFI System Partition. This partition must have mountpoint as \`/boot/efi'.\n\n
+EFI System Partition. This partition must have mountpoint as /boot/efi\n
+or /boot.\n\n
```

Comment:
Wrong.

Reply:
The terminology is wrong here.  The one that bricks your system is efivars, this is just the EFI location for boot.